### PR TITLE
Make October 2023 news public

### PR DESCRIPTION
**Pull request by cticenhour**
Add November 2023 template

Refs #11447

@idaholab/moose-ccb As usual, please enter in anything that you've added during October that didn't get added in a PR. 

@GiudGiud As I will be on PL next week, I am making this PR early. Could you remind folks and shepherd this along a bit next week for me?

### CODE REVIEW

**Comment by GiudGiud** [comment]
Will do

**Comment by GiudGiud** [comment]
@idaholab/moose-ccb last call for the newsletter

File: modules/doc/content/newsletter/2023/2023_10.md [hunk 1]
```diff
@@ -17,8 +12,80 @@ The following syntax can now be used to include other input files in an input fi
...
+intrinsically
+[LBB stable](https://en.wikipedia.org/wiki/Ladyzhenskaya%E2%80%93Babu%C5%A1ka%E2%80%93Brezzi_condition)
+and allows a natural upwind stabilization for momentum advection. For more
+information regarding the method, please see [hcgdgfe.md].
+
+### Module rename
+
+The Heat Conduction module was renamed to Heat Transfer to better match the capabilities of the module.
+
 ## libMesh-level Changes
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2023/2023_10.md:46]:
```suggestion
```
This section should be removed. As I don't think we've had a libmesh update this month - @roystgnr correct if wrong.

**Comment by grmnptr** [comment]
I'll add something about the SIMPLE solver

**Comment by cticenhour** [comment]
Blackbear docs failure is related to NEML2, not this PR. @loganharbour do you have time for a review?

**Comment by hugary1995** [comment]
FYI, the patch for blackbear docs is here: https://github.com/idaholab/blackbear/pull/367.

**Review by loganharbour — APPROVED** [review]

**Pull request merged**